dotnet/src/Functions/Functions.OpenAPI/OpenAI/SnakeCaseJsonNamingPolicy.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.Json;
+
+namespace Microsoft.SemanticKernel.Functions.OpenAPI.OpenAI;
+
+/// <summary>
+/// Available in .Net8:
+/// https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonnamingpolicy.snakecaselower?view=net-8.0#system-text-json-jsonnamingpolicy-snakecaselower
```

Comment:
You should be able to just use this now. We're now referencing the 8.0 version of System.Text.Json.